src/auth/models.cpp

```diff
@@ -101,6 +101,35 @@ std::string PermissionLevelToString(PermissionLevel level) {
   }
 }
 
+LabelPermission PermissionToLabelPermission(uint64_t permission) {
+  if (permission & LabelPermission::CREATE_DELETE) {
```

Comment:
Here a temporary object for which we are not really interested in is created, maybe use `&&` to produce boolean.